iceoryx_binding_c/source/c_publisher.cpp

```diff
@@ -99,11 +99,31 @@ void iox_pub_deinit(iox_pub_t const self)
     self->~cpp2c_Publisher();
 }
 
-iox_AllocationResult iox_pub_loan_chunk(iox_pub_t const self, void** const chunk, const uint32_t payloadSize)
+bool iox_pub_configure_custom_header(iox_pub_t const self,
+                                     const uint32_t customHeaderSize,
+                                     const uint32_t customHeaderAlignment)
 {
-    auto result = PublisherPortUser(self->m_portData)
-                      .tryAllocateChunk(payloadSize, CHUNK_DEFAULT_PAYLOAD_ALIGNMENT)
-                      .and_then([&chunk](ChunkHeader* h) { *chunk = h->payload(); });
+    auto settingsCheck = iox::mepoo::ChunkSettings::create(0U, 1U, customHeaderSize, customHeaderAlignment);
```

Comment:
Shouldn't `IOX_C_DEFAULT_USER_PAYLOAD_ALIGNMENT` be used here?

Reply:
could be used, but it's not necessary since I want to check if the custom header arguments are correct, so any payload alignment will do and `1U` was less to type ;)